dev.restfox.Restfox.yaml

```diff
@@ -0,0 +1,40 @@
+app-id: dev.restfox.Restfox
+base: org.electronjs.Electron2.BaseApp
+base-version: "24.08"
+command: restfox
+finish-args:
+  - --device=dri
+  - --filesystem=home
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+modules:
+  - name: restfox
+    build-commands:
+      - ar -o -x restfox.deb
+      - tar --extract --file data.tar.gz
+      - cp --preserve --recursive usr/* /app
+      - desktop-file-edit --set-key Exec --set-value "/app/bin/restfox %U" /app/share/applications/restfox.desktop
+      - install -D --mode 0644 /app/lib/restfox/resources/app/ui/favicon.svg /app/share/icons/hicolor/scalable/apps/dev.restfox.Restfox.svg
+      - install -D --mode 0644 --target-directory /app/share/metainfo/ dev.restfox.Restfox.metainfo.xml
+      - install -D --mode 0755 entrypoint.sh /app/bin/restfox # Overwrites sysmlink ¯\_(ツ)_/¯
+    buildsystem: simple
+    sources:
+      - type: file
+        path: dev.restfox.Restfox.metainfo.xml
+      - type: file
+        dest-filename: restfox.deb
+        sha256: 22e5434b69eff39d1c42ef503801b3c4aa578a4eeb26389ca20163ab32520585
+        url: https://github.com/flawiddsouza/Restfox/releases/download/v0.29.0/restfox_0.29.0_amd64.deb
+        # TODO: x-checker-data
+      - type: script
+        commands:
+          - export TMPDIR="${XDG_RUNTIME_DIR}/app/dev.restfox.Restfox"
+          - exec zypak-wrapper /app/lib/restfox/Restfox "$@"
+        dest-filename: entrypoint.sh
+rename-desktop-file: restfox.desktop
```

Comment:
these last line are misplaced and should be in the top section.

Reply:
I use `eslint-plugin-yml` for YAML, and it sorts they keys alphabetically by default.

Reply:
then don't. I mentionned it yesterday on Matrix they were misplaced. Does this sort the javascript line? I don't think so.